inst/app/app.R

```diff
@@ -0,0 +1,219 @@
+library(shiny)
+library(shinyWidgets)
+library(shinyjs)
+library(shinybusy)
+library(ggplot2)
+library(dplyr)
+library(rplanes)
+library(lubridate)
+
+# list module files and iterate sourcing them to use within the app.
+module_sources <- list.files(path = here::here("inst/app/modules"), full.names = TRUE)
+#module_sources <- list.files(path = system.file("app/modules/", package = "rplanes"), full.names = TRUE)
+sapply(module_sources, source)
+
+
+# UI SIDE ####
+
+ui <- fluidPage(
+    useShinyjs(),  # Set up shinyjs
+    titlePanel("Rplanes Explorer"),
+    sidebarLayout(position = "left",
+                  sidebarPanel(width = 3,
+                      prettyRadioButtons("choice", "Choose Dataset", choices = c("Example", "Custom"), status = "warning", inline = TRUE, icon = icon("check"), bigger = TRUE),
+                      shinyjs::hidden(div(id = "choice_custom",
+                                          fileInput("upload_1", label = "Upload Observed Data", multiple = F, accept = ".csv"),
+                                          fileInput("upload_2", label = "Upload Comparison", multiple = F, accept = ".csv")
+                                          )),
+                                          h6("Is the Comparison a Forecast"),
+                                          prettyToggle("status", label_on = "Yes", icon_on = icon("check"), status_on = "success", label_off = "No", icon_off = icon("remove"), status_off = "warning", value = TRUE),
```

Comment:
the "prettyToggle" looks nice ... but it is a little confusing. and it obscures what the "Not Forecast" condition actually means. i would prefer just radioButtons with a label "Type of signal to be evaluated" and options ("Forecast", "Observed") ... and of course those input choices will need to propagate to server logic